02.calendar/my-calelndar.js

```diff
@@ -0,0 +1,50 @@
+"use strict";
+const { DateTime } = require("luxon");
+const minimist = require("minimist");
+
+const options = minimist(process.argv.slice(2), {
+  year: "y",
+  month: "m",
+  default: {
+    year: DateTime.now().year,
+    month: DateTime.now().month,
+  },
+});
+
+const year = options.y;
+const month = options.m;
+
+const dt = DateTime.fromObject({
+  year: year,
+  month: month,
+  day: 1,
+});
+
+if (dt.month < 10) {
+  console.log(`      ${dt.month}月 ${dt.year}`);
+} else {
+  console.log(`     ${dt.month}月 ${dt.year}`);
+}
+
+console.log("日 月 火 水 木 金 土");
+const endDt = dt.endOf("month");
+
+if (dt.weekday < 7) {
+  const RawSpace = " ";
+  const MakingSpace = RawSpace.repeat(3 * dt.weekday);
```

Comment:
これも小文字始まりでOKです。
ただ、RawSpaceとMakingSpaceは変数名として少し分かりにくい気がします。

```suggestion
  const space = " ".repeat(3 * dt.weekday);
```

みたいな感じでも十分かもしれません。

Reply:
ご教示いただいた例の方が分かりやすく、こちらに合わせて修正いたしました！